dev/dags/dbt/model_version/README.md

```diff
@@ -0,0 +1,11 @@
+## `jaffle_shop`
```

Comment:
Could we have a smaller dbt sample project to illustrate the version feature, or do we need to stick to the jaffle shop one?

Reply:
It big because, I want to make sure this PR can cover the use case:
> The second expectation is the dag will render right depedences, for example: stg_customers_v1 >> customers_v1, stg_customers_v2 >> customer_v2...
Oh, it is lack of test for those dependencies. => Will add soon.

About stick to the `jaffle_shop` idea, I think we will get dbt version problem for existing tests? Because this feature only work with dbt>=1.6.0. It is my opinion, if I am wrong, please correct me :'(